pkg/podmanstructs/podmanstructs.go

```diff
@@ -0,0 +1,204 @@
+package podmanstructs
```

Comment:
Could you rename this to inspectstructs? Or inspecttypes?

Reply:
Maybe even plain old "inspect" to keep things simple

Reply:
Yes I like that.

Reply:
fixed